crates/ruma-common/Cargo.toml

```diff
@@ -53,7 +53,7 @@ as_variant = { workspace = true }
 base64 = { workspace = true }
 bytes = { workspace = true }
 form_urlencoded = "1.0.0"
-getrandom = { version = "0.2.6", optional = true }
+getrandom = { version = "0.3.1", optional = true }
```

Comment:
It looks like WASM support for getrandom requires to set a `RUSTFLAG` now… Should we document it somewhere?

If we don't enable the feature ourselves, we don't need need the dependency.